CHANGELOG.md

```diff
@@ -23,6 +35,20 @@
 
 * `ilab`: **Deprecation of Python 3.9 support and withdrawal of Python 3.12 support** Due to changes to training requiring the usage of [GPTDolomite](https://github.com/instructlab/GPTDolomite), Python 3.9 is no longer supported and Python 3.12 support is currently withdrawn. If you are using either of these versions, you will need to start using either Python 3.10 or Python 3.11 to use this and subsequent versions of the CLI.
 * `ilab model train`: The '--device' parameter no longer supports specifying a GPU index (e.g., 'cuda:0'). To use a specific GPU, set the visible GPU before running the train command.
+* `ilab init`: With the introduction of a dedicated storage system within the `ilab` CLI,
+   `ilab init` and `ilab config init` will now output and read the config file from the
+   platform's config directory under the `instructlab` package.
+* `ilab sysinfo`: This command has been moved to now exist as `ilab system info`.
```

Comment:
That is not a breaking change. I have added an alias.

Reply:
Gotcha, I'll remove it.